scripts/initReport.mjs

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env node
+
+/**
+ * This script creates a new report file with the current date on a new branch.
+ */
+
+import fs from 'fs'
+import { fileURLToPath } from 'url'
+import { execSync } from 'child_process'
+
+const REPORTS_DIRECTORY = 'content/reports'
+const REPORT_TYPE = 'contributors'
+const PULLS_URL =
+  'https://api.github.com/search/issues?q=is:pr+user:TOSIT-IO+is:open+sort:updated-asc'
+
+function formatCurrentDate(date) {
+  const year = date.getFullYear().toString().slice(-2)
+  const month = (date.getMonth() + 1).toString().padStart(2, '0')
+  const day = date.getDate().toString().padStart(2, '0')
+  return `${year}${month}${day}`
+}
+
+function checkoutNewBranch(branchName) {
+  try {
+    const currentBranch = execSync('git branch --show-current')
+      .toString()
+      .trim()
+
+    if (currentBranch !== branchName) {
+      try {
+        execSync(`git checkout ${branchName}`)
+      } catch {
+        execSync(`git checkout -b ${branchName}`)
+      }
+    }
+  } catch (error) {
+    console.error('Git error:', error)
+    console.error('Failed to checkout new branch')
+  }
+}
+
+async function fetchPulls() {
+  try {
+    const pulls = await fetch(PULLS_URL)
+    return pulls.json()
+  } catch (error) {
+    console.error('Failed to fetch pulls:', error)
+    return []
+  }
+}
+
+function formatPulls(pulls) {
+  if (!pulls || pulls.total_count === 0) {
+    return ''
+  }
+  return pulls
+    .map((pull) => {
+      const { number, title, url } = pull
+      const repo = pull.repository_url.split('/').pop()
+
+      return `- [${repo}#${number}](${url}): <!-- status of "${title}" -->`
+    })
+    .join('\n')
+}
+
+function generateReportContent(pullsDetails, currentDate) {
+  return `---
+type: ${REPORT_TYPE}
+date: ${currentDate.toISOString().split('T')[0]}
+description: |
+<main topics of the meeting>
+---
+
+# TDP ${REPORT_TYPE} meeting notes
+
+<!-- general notes -->
+
+## Pull Requests
+
+Weekly review of open PRs (in chronological order):
+
+${pullsDetails}
+
+## Open topics
+
+<!-- open topics -->
+`
+}
+
+async function main() {
+  try {
+    const rootDirURL = new URL('..', import.meta.url)
+    const repoPath = fileURLToPath(rootDirURL)
+
+    const currentDate = new Date()
+    const dateStr = formatCurrentDate(currentDate)
+
+    const newBranchName = `${dateStr}-${REPORT_TYPE}-report`
+    process.chdir(repoPath)
+    checkoutNewBranch(newBranchName)
+
+    const pulls = await fetchPulls()
+
+    if (!pulls || pulls.total_count === 0) {
+      console.error('No pull requests to report')
+    }
+
+    const pullsDetails = formatPulls(pulls.items)
+    const reportContent = generateReportContent(pullsDetails, currentDate)
+
+    const reportName = `${dateStr}-${REPORT_TYPE}.md`
+    const reportsDirURL = new URL(REPORTS_DIRECTORY, rootDirURL)
+    const reportsDirPath = fileURLToPath(reportsDirURL)
+    const reportPath = `${reportsDirPath}/${reportName}`
```

Comment:
Report in both singular and plural forms, I would rename `reports...` with `reportBase...`